### PR TITLE
Doc/combinatorial testing

### PR DESCRIPTION
**Pull request by muratkeremozcan**
added a section about combinatorial testing with simple examples

### CODE REVIEW

**Comment by muratkeremozcan** [comment]
@NoriSte @anoop-gupt 
Hey guys, please take a look and critique if it makes sense to a layman, enough to get them started.

File: README.md [hunk 1]
```diff
@@ -222,6 +222,15 @@ If a test cannot produce reliable results, it reduces confidence in the tests an
 🔗 [**Read More: Test Flake**](./sections/advanced/test-flake.md)
 
 
+<br/>
+
+## ![✔] 6.3 Combinatorial Testing
+
+**TL;DR:** Most sw bugs and failures are caused by one or two parameters. Testing parameter combinations can provide more efficient fault detection than conventional methods. Combinatorial Testing is a proven method for more effective sw testing at a lower cost.
```
Comment (anoop-gupt) [inline README.md]: sw would be confusing - use "software"

**Comment by NoriSte** [comment]
Thank you so much @muratkeremozcan  and sorry for the big delay. Take a look at the changes I made and check if they are fine from your side.

https://foselab.unibg.it/ctwedge/ is unavailable at the time of writing but I think it's a temporary issue...

Anyway: really interesting, I share it both internally and publicly!!

**Comment by muratkeremozcan** [comment]
Rebased and resolved merge conflict.

> Thank you so much @muratkeremozcan and sorry for the big delay. Take a look at the changes I made and check if they are fine from your side.
> 
> https://foselab.unibg.it/ctwedge/ is unavailable at the time of writing but I think it's a temporary issue...
> 
> Anyway: really interesting, I share it both internally and publicly!!

**Pull request merged**